docs/reference/redirects.asciidoc

```diff
@@ -1930,4 +1930,18 @@ coming::[8.10.0]
 [role="exclude",id="put-synonyms"]
 === Create or update synonym sets API
 
-coming::[8.10.0]
\ No newline at end of file
+coming::[8.10.0]
+
+ifeval::["{release-state}"=="released"]
+
+[role="exclude",id="security-api-create-cross-cluster-api-key"]
+=== Create Cross-Cluster API key API
+
+{es-feature-flag}
```

Comment:
The text for this attribute doesn't really make sense here:

> This feature is in development and not yet available for use. This documentation is provided for informational purposes only.

There is no "This documentation..."

Reply:
True. I thought that was a better stop-gap than referring to master docs for now, but hopefully there will be a quick fix and we can remove these redirects entirely.